src/commands/Minion/autoslay.ts

```diff
@@ -307,6 +307,15 @@ export default class extends BotCommand {
 				return masterMatch && e.monsterID === usersTask.assignedTask!.monster.id;
 			});
 
+			const ehpKillable = killableMonsters.find(m => m.id === ehpMonster?.efficientMonster);
+
+			// If we don't have the requirements for the efficient monster, revert to default monster
+			if (
+				ehpKillable?.levelRequirements !== undefined &&
+				!msg.author.hasSkillReqs(ehpKillable.levelRequirements)[0]
+			)
+				return this.client.commands.get('k')?.run(msg, [null, usersTask.assignedTask!.monster.name]);
```

Comment:
if an `if` statement is longer than 1 line, it needs to always have parens `{}`, its not enforced by eslint yet.